src/FlashList.tsx

```diff
@@ -382,7 +398,10 @@ class FlashList<T> extends React.PureComponent<
           itemAnimator={this.itemAnimator}
           suppressBoundedSizeException
           externalScrollView={
-            renderScrollComponent as RecyclerListViewProps["externalScrollView"]
+            this.props.experimentalScrollPositionManagement &&
+            Platform.OS === "android"
```

Comment:
Use the filers in config folder to add platform check. We discourage inline platform checks.